### PR TITLE
DIV-4579 create super user

### PR DESCRIPTION
**Pull request by hosainnet**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines

### JIRA link (if applicable) ###
DIV-4579

### Change description ###

- Added new role `caseworker-divorce-superuser` (`AuthorisationCaseType.json`)
- Role has access to all existing fields (`AuthorisationCaseField.json`)
- Role has access to all existing events (`AuthorisationCaseEvent.json`)
- Role has access to all existing states (`AuthorisationCaseState.json`)
- Role has access to new events, allowing them to move case from any state `*` to all of our states (`CaseEvent.json`)
- Added some test users to `UserProfile.json`

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: definitions/divorce/json/AuthorisationCaseEvent.json [hunk 1]
```diff
@@ -1825,5 +1825,1349 @@
...
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "aosReceivedNoAdConStarted",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "aosReceivedNoAdConOverdue",
```
Comment (alexwakeman) [inline definitions/divorce/json/AuthorisationCaseEvent.json]: Why do we have to repeat these events? I thought any super-user would have the other court-admin roles assigned to them as well, like `courtadmin_beta`?

Comment (alexwakeman) [reply definitions/divorce/json/AuthorisationCaseEvent.json]: i.e. why repeat permissions?

Comment (hosainnet) [reply definitions/divorce/json/AuthorisationCaseEvent.json]: I think the idea is to have them self-sufficent and not relay on compound roles - that's what I gathered from https://tools.hmcts.net/jira/browse/DIV-4579
@gavinclark  can confirm though..

Comment (alexwakeman) [reply definitions/divorce/json/AuthorisationCaseEvent.json]: Seems un-necessary when they could have the permissions through the existing role? That's kind of the point of having this ability in CCD. It becomes error prone in terms of maintaining permissions across both user role types as well, each time a new event is introduced.

Comment (hosainnet) [reply definitions/divorce/json/AuthorisationCaseEvent.json]: Fair enough, hopefully @gavinclark can chip in soon!

Comment (hosainnet) [reply definitions/divorce/json/AuthorisationCaseEvent.json]: Updated as per discussion on DIV-4579

File: definitions/divorce/json/CaseTypeTab.json [hunk 2]
```diff
@@ -1860,7 +1860,7 @@
     "TabDisplayOrder": 3,
     "CaseFieldID": "D8caseReference",
     "TabFieldDisplayOrder": 1,
-    "TabShowCondition": "D8ReasonForDivorce=\"adultery\" AND (D8ReasonForDivorceAdulteryWishToName=\"Y*\" OR D8ReasonForDivorceAdulteryIsNamed=\"Y*\")"
+    "TabShowCondition": "D8ReasonForDivorce=\"adultery\" AND D8ReasonForDivorceAdulteryWishToName=\"Y*\""
```
Comment (hosainnet) [inline definitions/divorce/json/CaseTypeTab.json]: @mounikahmcts  FYI ^

**Pull request merged**